src/Nethermind/Nethermind.TxPool/TxPoolConfig.cs

```diff
@@ -21,6 +21,7 @@ public class TxPoolConfig : ITxPoolConfig
         public int HashCacheSize { get; set; } = 512 * 1024;
         public long? GasLimit { get; set; } = null;
         public long? MaxTxSize { get; set; } = 128.KiB();
+        public long? MaxBlobTxSize { get; set; } = 1.MiB();
```

Comment:
why blopb tx's can be 8x bigger?

Reply:
Doesn't have sense to me, but I wanted to do it similar to Geth implementation
https://github.com/ethereum/go-ethereum/blob/master/core/txpool/blobpool/blobpool.go#L63-L68

Reply:
And it is not a big deal as blob txs are saved to disc. Memory is not affected as in case of non-blob txs

Reply:
We can also go with standard 128 KiB and it will simplify this PR, but we will then reject some txs accepted by Geth